Library/ViewModels/PledgeViewModel.swift

```diff
@@ -810,20 +810,45 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
 
     self.title = context.map { $0.title }
 
+    let trackCheckoutPageViewData = Signal
+      .zip(
+        project,
+        baseReward,
+        rewards,
+        selectedQuantities,
+        refTag,
+        initialAdditionalPledgeAmount,
+        pledgeTotal,
+        baseRewardShippingTotal,
+        context.filter { $0 == .pledge }
```

Comment:
this is a good idea but let's clean it up a bit. try this out and test the event on the Segment debugger to verify it's working for new pledges and not for changing payment method.
just `context` here